### ISSUE TITLE
Higher Resolution favicon

### ISSUE THREAD

**Issue by JosephShepin** [PrairieLearn/PrairieTest-feedback#43]
I use Prarielarn so much, I have a shortcut on my computer to get there fast but the icon looks a little...blurry. It would be great to have a higher resolution asset there because I can see the distortion and artifacts pretty clearly.

<img width="238" alt="Screenshot 2023-11-27 at 11 24 51 PM" src="https://github.com/PrairieLearn/PrairieTest-feedback/assets/31457361/d918db1d-7f30-4181-a368-82a0396e6ec6">

**Comment by nwalters512** [comment]
I agree that this looks pretty bad! I'm going to transfer this to the PrairieLearn repo since this relates to PL (although PT almost definitely has the same issue).
